tests/generic_config_updater/test_eth_interface.py

```diff
@@ -322,8 +323,9 @@ def test_replace_fec(duthosts, rand_one_dut_front_end_hostname, ensure_dut_readi
                           "Failed to properly configure interface FEC to requested value {}".format(fec))
 
             # The rollback after the test cannot revert the fec, when fec is not configured in config_db.json
```

Comment:
Hi @wen587, is this expected behavior or it's image issue? It sounds weird that GCU can't rollback fec.

Reply:
Hi @lizhijianrd , I think the issue is that when fec is not configured before the test, the rollback is jut remove the fec field instead of configure it to previous state.
If the fec is already configured before test, then it will know how to set it to the init state.